examples/nat64/main.rs

```diff
@@ -150,28 +150,20 @@ fn nat_6to4(ethernet: Ethernet) -> Fallible<Either<Ethernet>> {
     }
 }
 
-fn install(qs: HashMap<String, PortQueue>) -> impl Pipeline {
+fn install_6_to_4(qs: HashMap<String, PortQueue>) -> impl Pipeline {
     Poll::new(qs["eth1"].clone())
         .map(|packet| packet.parse::<Ethernet>())
```

Comment:
this map can be moved into the nat fn. alternatively move the ipv6 parse out of that fn. it's a little inconsistent to be parsing in 2 different places like this.

Reply:
+1